docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:18.04
+
+COPY requirements.txt requirements.txt
+
+RUN apt-get	update && \
+	apt-get install -y --no-install-recommends apt-utils python3 python3-pip sqlite3 fftw3 python3-setuptools && \
+	apt-get install -y --no-install-recommends python3-dev libfftw3-dev git && \
+	pip3 install --no-cache-dir -r requirements.txt && \
+	apt-get remove -y python3-dev libfftw3-dev git && \
+	apt-get -y autoremove && \
+	rm -rf /var/lib/apt/lists/*
+
+
+## set cesium and mabox keys
```

Comment:
Can you remove these comments?

Reply:
Yes, of course. Comments in bottom are corrected according to the Dockerfile.